viz_config.yml

```diff
@@ -71,7 +72,7 @@ title_cfg:
   subtitle: "Water Year 2018"
   footnote: [
     "Conditions are relative to the historic daily record for each gage.",
-    "Flood status was readily available for 17% of sites when this graphic was produced."
+    "Flood status available for 38% of sites when this graphic was produced."
```

Comment:
:+1: nice and concise. I might include the "was" just so the style is consistent with the other footnote.